### ISSUE TITLE
Microphone keeps recording after clicking the "stop" button

### ISSUE THREAD

**Issue by wodim** [MediaCrush/MediaCrush#666]

**Comment by jleclanche** [comment]
Shit, they found the backdoor. Run!

**Comment by wodim** [comment]
mfw #666

**Comment by jleclanche** [comment]
Literally not a coincidence

**Comment by SirCmpwn** [comment]
Yeah, but it doesn't keep "recording" into an upload. This isn't really the wrong behavior. Feel free to send a PR that changes this but I don't think anyone really cares.

**Comment by wodim** [comment]
I can understand you do not want to fix it, but this is the wrong behaviour, since 1) the webpage is still recording even if it is not going to upload that part, 2) Chrome shows a red dot in the tab list which will confuse users, 3) Chrome (via Windows, I guess) lowers the volume of anything that is playing while the webpage it is still recording.

**Comment by jleclanche** [comment]
@SirCmpwn He's right, there's a lot of reason why this should be fixed. Battery usage as well for example.

**Comment by SirCmpwn** [comment]
What I'm saying is send a pull request cause I'm not doing it.

**Comment by wodim** [comment]
No. You said that it was not the wrong behaviour and that users do not care about it.
